docs/_docs/quickstart.md

```diff
@@ -4,25 +4,37 @@ title: Quick-start guide
 permalink: /docs/quickstart/
 ---
 
-For the impatient, here's how to get a boilerplate Jekyll site up and running.
+If you already have [Ruby](https://www.ruby-lang.org/en/downloads/) and [RubyGems](https://rubygems.org/pages/download) installed (see Jekyll's [requirements](/docs/installation/#requirements/)), you can create a new Jekyll site by doing the following:
 
 ```sh
-~ $ gem install jekyll bundler
-~ $ jekyll new myblog
+# Install Jekyll and Bundler through RubyGems
+~ $ gem install jekyll bundler 
+
+# Create a new Jekyll site at ./myblog
+~ $ jekyll new myblog 
+
+# Change into your new directory
 ~ $ cd myblog
-~/myblog $ bundle exec jekyll serve
-# => Now browse to http://localhost:4000
+
+# Build the site on the preview server
+~/myblog $ bundle exec jekyll serve 
+
+# Now browse to http://localhost:4000
 ```
 
-The `jekyll new` command now automatically initiates `bundle install` and installs the dependencies required. To skip this, pass `--skip-bundle` option like so `jekyll new myblog --skip-bundle`.
+`gem install jekyll bundler` installs the [jekyll](https://rubygems.org/gems/jekyll/) and [bundler](https://rubygems.org/gems/bundler) gems through [RubyGems](https://rubygems.org/). You need only to install the gems one time &mdash; not every time you create a new Jekyll project. Here are some additional details:
+
+* `bundler` is a gem that manages other RubyGems. It makes sure your gems and gem versions are compatible, and that you have all necessary dependencies each gem requires. 
+* The Gemfile and Gemfile.lock files inform Bundler about the gem requirements in your theme. If your theme doesn't have these Gemfiles, you can omit `bundle exec` and just run `jekyll serve`.
+
+* When you run `bundle exec jekyll serve`, Bundler uses the gems and versions as specified in Gemfile.lock to ensure your Jekyll site builds with no compatibility or dependency conflicts.
```

Comment:
why are we setting off file names in code formatting? code tags are for code, right? i'll update for now but want to look at all usage across the site.